README.md

```diff
@@ -48,8 +48,13 @@ params:
   region: us-east-1
   accountId: <from above>
   kubeWorkerAMI: ami-<from above>
-  productionHost: domain.com
-  stagingHost: domain-staging.com
+  productionHostRoot: domain.com
+  productionFrontendHost: domain.com
+  productionBackendHost: api.domain.com
+  stagingHostRoot: domain-staging.com
+  stagingFrontendHost: domain-staging.com
+  stagingBackendHost: api.domain-staging.com
```

Comment:
More like "lets talk about this later" but this is the kind of place where we'll benefit by having these configuration values but also providing reasonable defaults too.

Reply:
I think these are already good suggestions. Use different domains for staging and production, have the front end served from the root, provide your backend on a subdomain called `api`